third_party/proton/proton/viewer.py

```diff
@@ -187,6 +187,15 @@ def parse(metrics, filename, include, exclude, threshold, depth, format):
         # TODO: generalize to support multiple metrics, not just the first one
         gf = filter_frames(gf, include, exclude, threshold, metrics[0])
         print(gf.tree(metric_column=metrics, expand_name=True, depth=depth, render_header=False))
+        emitWarnings(gf, metrics)
+
+
+def emitWarnings(gf, metrics):
+    if ("bytes (inc)" in metrics):
```

Comment:
Why cannot we make it `if "bytes (inc)" in metrics`:?

Reply:
OK. Updated.